packages/app-project/src/components/ProjectHeader/helpers/getNavLinks/getNavLinks.spec.js

```diff
@@ -60,7 +62,44 @@ describe('Helper > getNavLinks', function () {
       const links = getNavLinks(true, BASE_URL, '1234')
       expect(links).to.deep.equal([
         {
-          href: `${BASE_URL}/about`,
+          href: `${BASE_URL}/about/research`,
+          text: 'About'
+        },
+        {
+          href: `${BASE_URL}/classify/workflow/1234`,
+          text: 'Classify'
+        },
+        {
+          href: `${BASE_URL}/talk`,
+          text: 'Talk'
+        },
+        {
+          href: `${BASE_URL}/collections`,
+          text: 'Collect'
+        },
+        {
+          href: `${BASE_URL}/recents`,
+          text: 'Recents'
+        }
+      ])
+    })
+  })
+
+  describe('when PANOPTES_ENV is production', function () {
+    let previousEnv
+    before(function () {
+      previousEnv = process.env.PANOPTES_ENV
+      process.env.PANOPTES_ENV = 'production'
```

Comment:
In other tests, we use `proxyquire` to mock the Node environment.
https://github.com/zooniverse/front-end-monorepo/blob/debb99711151ce9a1e28d56790bd3a2fe64b44f2/packages/app-project/server/set-logging.spec.js#L12-L20